django_project/base/static/css/landing_page.css

```diff
@@ -150,16 +150,16 @@ header.masthead p {
     text-align: left;
 }
 
-#search {
+#search1 {
     border: none;
     height: 30px;
 }
 
-#search:focus {
+#search1:focus {
```

Comment:
why renaming this @ann26 ?
maybe rename it with good word? like search-on-map?

Reply:
hi @meomancer it's because I renamed the element in carousel.html so that it won't conflict the element with the same name in map.html. Ok I will change the name.